content/tutorials/create-users-and-teams.md

```diff
@@ -160,38 +164,43 @@ However, there are times when you need to configure permissions on a more granul
 
 ### Exercise
 
-Graphona has hired a consultant to assist the Marketing team. The consultant should only be able to access the _SEO_ dashboard in the _Analytics_ folder.
+Graphona has hired a consultant to assist the Marketing team. The consultant should only be able to access the SEO dashboard in the Analytics folder.
 
 | Name       | Email                            | Username   |
 |------------|----------------------------------|------------|
 | Luc Masson | luc.masson@exampleconsulting.com | luc.masson |
 
 #### Add a new user
 
-1. In the sidebar, click **Server Admin** -> **Users**.
-1. Click **New user**.
-1. In the **Name** box, enter the name of the user.
-1. In the **Email** box, enter the email of the user.
-1. In the **Username** box, enter the username that the user will use to log in.
-1. In the **Password** box, enter a password. The user will be able to change 1.heir password once they log in.
-1. Click **Create** to create the user.
+1. On the sidebar, click the **Server Admin** (shield) icon.
```

Comment:
```suggestion
1. In the sidebar, click the **Server Admin** (shield) icon.
```

I assume this is a typo, otherwise we'd have to change all of these?